samples/python/console/speech_sample.py

```diff
@@ -37,6 +37,8 @@
 weatherfilenamemp3 = "whatstheweatherlike.mp3"
 weatherfilenamemulaw = "whatstheweatherlike-mulaw.wav"
 seasonsfilename = "pronunciation_assessment_fall.wav"
+zhcnfilename = "zhcn_short_dummy_sample.wav"
+zhcnlongfilename = "zhcn_long_dummy_sample.wav"
```

Comment:
maybe `zhcnlongfilename = "zhcn_continuous_mode_sample.wav"` is better.

Reply:
updated.